components/inlinealert/index.css

```diff
@@ -120,42 +191,30 @@
 
 .spectrum-InLineAlert-footer {
 	display: flex;
-	justify-content: flex-end;
-	margin-block-start: var(--mod-inlinealert-spacing-header-content-button, var(--spectrum-inlinealert-spacing-header-content-button));
+	justify-content: flex-start;
+	margin-block-start: var(--mod-inlinealert-spacing-content-link-button, var(--spectrum-inlinealert-spacing-content-link-button));
 
 	&:empty {
 		display: none;
 	}
 }
 
-.spectrum-InLineAlert--info {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-info, var(--spectrum-inlinealert-border-and-icon-color-info)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-info, var(--spectrum-inlinealert-border-and-icon-color-info)));
-	}
-}
-
-.spectrum-InLineAlert--notice {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-notice, var(--spectrum-inlinealert-border-and-icon-color-notice)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-notice, var(--spectrum-inlinealert-border-and-icon-color-notice)));
-	}
-}
-
-.spectrum-InLineAlert--positive {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-positive, var(--spectrum-inlinealert-border-and-icon-color-positive)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-positive, var(--spectrum-inlinealert-border-and-icon-color-positive)));
-	}
-}
-
-.spectrum-InLineAlert--negative {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-negative, var(--spectrum-inlinealert-border-and-icon-color-negative)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-negative, var(--spectrum-inlinealert-border-and-icon-color-negative)));
+@media (forced-colors: active) {
+	.spectrum-InLineAlert {
+		--highcontrast-inlinealert-header-color: CanvasText;
+		--highcontrast-inlinealert-content-color: CanvasText;
+		--highcontrast-inlinealert-border-and-icon-color: CanvasText;
+
+		.spectrum-InLineAlert-header,
+		.spectrum-InLineAlert-content,
+		.spectrum-InLineAlert-footer {
+			forced-color-adjust: none;
+		}
+
+		&.spectrum-InLineAlert--subtle,
+		&.spectrum-InLineAlert--bold {
+			--highcontrast-inlinealert-background-color: Canvas;
+			--mod-inlinealert-border-width: var(--spectrum-inlinealert-border-width);
```

Comment:
Suggestion: I'd add `--highcontrast-inlinealert-border-width` to the stack where it's used in the `border-inline-width`, and then set the value here with `--highcontrast-inlinealert-border-width: var(--spectrum-border-width-200);`